src/compiler/WebSharper.FSharp.Service/Program.fs

```diff
@@ -30,7 +31,44 @@ open WebSharper.Compiler
 open WebSharper.Compiler.FSharp.Compile
 open WebSharper.Compiler.FSharp.ErrorPrinting
 open WebSharper.Compiler.CommandTools
+open System.Collections.Generic
 
+let argsDict = Dictionary<string, string []>()
+
+let (|Exit|FullCompile|PostCompile|) (args: ArgsType) = 
+    let tryGetDirectoryName (path: string) =
+        try
+           System.IO.Path.GetDirectoryName path |> Some
+        with
+        | :? System.DivideByZeroException -> None
```

Comment:
This looks weird, when is GetDirectoryName throwing DivideByZeroException? It's not documented.

Reply:
I had an unfinished code part checking if the directory exists. Good catch